lib/capistrano-deploy/hipchat.rb

```diff
@@ -5,23 +5,67 @@ def self.load_into(configuration)
         namespace :hipchat do
           require 'hipchat'
 
-          set(:deployer) { %x(git config user.name).chomp }
+          set(:deployer) { human }
+
           desc 'Set notification in hipchat for deployment start'
           task :start_msg do
             client = HipChat::Client.new(hipchat_token)
-            deploy_text = "#{deployer} is starting deploy of '#{app_name.upcase}' from branch '#{branch.upcase}' to #{current_stage.upcase}"
-
+            deploy_text = "#{human} is deploying #{deployment_name} to #{environment_string}."
             client[hipchat_room].send('Deploy', deploy_text, color: 'yellow')
           end
 
           desc 'Set notification in hipchat for deployment end'
           task :end_msg do
             client = HipChat::Client.new(hipchat_token)
-            deploy_text = "#{deployer} completed the deploy of '#{app_name.upcase}' from branch '#{branch.upcase}' to #{current_stage.upcase}"
+            deploy_text = "#{human} finished deploying #{deployment_name} to #{environment_string}."
 
             client[hipchat_room].send('Deploy', deploy_text, color: 'green')
           end
+
+          def human
+            user = ENV['HIPCHAT_USER'] || fetch(:hipchat_human, %x(git config user.name).chomp)
+            user = user || if (u = %x{git config user.name}.strip) != ''
```

Comment:
hmm...i wasn't sure how much i should change it since I mostly copy/pasted (with enough changes to get it to work with our stuff) from https://github.com/hipchat/hipchat-rb/blob/master/lib/hipchat/capistrano/tasks/hipchat.rake